GooglePlacesAutocomplete.js

```diff
@@ -556,18 +606,33 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
         setStateText(props.preProcess(text));
       }
 
-      request.open(
-        'GET',
-        `${url}/place/autocomplete/json?input=` +
-          encodeURIComponent(text) +
-          '&' +
-          Qs.stringify(props.query),
-      );
+      if (props.isNewPlacesAPI) {
+        request.open('POST', `${url}/v1/places:autocomplete`);
```

Comment:
If a key was passed in the library invocation, it needs to be added to the URL as a parameter. I know this is counterintuitive, as the rest of the data will be sent in the request body, but I tested and it works. The alternative is to send it in a header, but I think the query parameter is simpler. 

If you send it in the request body, the request will fail. That's fine for us, since our proxy strips it out, but it will not work for anyone using this library without a proxy.

Reply:
We now pass `key` as query param.